staging/src/k8s.io/apiserver/pkg/audit/context.go

```diff
@@ -65,8 +62,7 @@ type annotation struct {
 // prefer AddAuditAnnotation over LogAnnotation to avoid dropping annotations.
```

Comment:
This godoc mentions functions that no longer exist, please fix that. Perhaps the changes from https://github.com/kubernetes/kubernetes/pull/117406 might be applicable here?